core/src/util/usLDAPExpr.cpp

```diff
@@ -0,0 +1,852 @@
+/*=============================================================================
```

Comment:
Some files like `usLDAPExpr.cpp` are still present in their original location (core/src/service). I guess this is the reason why the changes are not schown as a move. Could you remove the files, squash the commits and push-force the branch again?